src/Eval.ml

```diff
@@ -111,6 +130,15 @@ let extern_map =
     "dbl_printInt",   int_fun (fun n -> print_int n; v_unit);
     "dbl_readLine",   unit_fun (fun () -> VStr (read_line ()));
     "dbl_exit",       int_fun exit;
+    "dbl_abstrType",  VFn (fun _ cont -> cont (VCtor(0, [])));
```

Comment:
I don't know if it was authors intention, but that `VCtor(0, [])` looks a lot like `v_unit` defined in line 78. If not maybe it's a good idea to create similar variable, something like `v_abstr` for better code documentation.

Reply:
It was my intention. Of course, we don't want to write `v_unit` because it is not the unit value. But I agree that defining a new variable for better code documentation is a good idea.